src/Resolver.js

```diff
@@ -392,6 +392,11 @@ class Resolver {
   }
 
   lookupAlias(aliases, filename, dir) {
+    filename = path.normalize(filename);
+    Object.keys(aliases).forEach(
+      key => (aliases[path.normalize(key)] = aliases[key])
+    );
```

Comment:
Probably not a good idea to mutate the package.json... Maybe do the normalization + checking in the existing for loop below instead

Reply:
Probably a better idea, will change it